priv/repo/migrations/20230508100505_add_admin_level_to_users.exs

```diff
@@ -0,0 +1,19 @@
+defmodule PetStore.Repo.Migrations.AddAdminLevelToUsers do
+  use Ecto.Migration
+
+  def up do
```

Comment:
I think you can just `s/up/change/g` and Elixir will figure out how to reverse the changes. Also, we usually simply use `add` when greenfielding, the migration system should take care of checking that the migration is already applied or not.

Reply:
Unfortunately that was my starting point, but trying to `mix ecto.rollback -n 1` I get 
```
12:22:36.434 [info] drop constraint admin_level_range from table users
** (Ecto.MigrationError) cannot reverse migration command: alter table users. You will need to explicitly define up/0 and down/0 in your migration
```

I removed the `_if_exists` and `_if_not_exists` as requested in 4d1b65a80a3e7666dbb26d7db794bdf89de7796d

Reply:
Seems like using `add` instead of `add_if_not_exists`, the rollback works just fine. I merged the two functions as requested